content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Aplicando os Padrões de Segurança do Pod, Configurando o Controlador de Admissão Embutido
+content_type: task
+min-kubernetes-server-version: v1.22
+---
+
+Desde a versão v1.22, o Kubernetes fornece um [controlador de admissão]
+(/docs/reference/access-authn-authz/admission-controllers/#podsecurity)
+embutido para fazer cumprir os [padrões de segurança do Pod]
+(/docs/concepts/security/pod-security-standards).
+Você pode configurar esse controlador de admissão para definir padrões em todo 
+o cluster e [excessões](/docs/concepts/security/pod-security-admission/#exemptions).
+
+## {{% heading "prerequisites" %}}
+
+{{% version-check %}}
+
+- Garanta que a `PodSecurity` do [portal de funcionalidade]
+(/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) 
+está ativada.
```

Comment:
This link is also broken. Markdown syntax does not allow for breaking lines between `[]` and `()`.

Another thought: I think we should refrain from translating feature gate. It would be better to keep it here but format in italic.